website/prebuild.py

```diff
@@ -88,7 +88,13 @@ def label_subdir(
     readme_titles = [readme_title]
     if parent_title:
         readme_titles.insert(0, parent_title)
-    children = [x for x in subdir.glob("**/*.md") if x != readme]
+    children = [
+        x
+        for x in subdir.glob("**/*.md")
+        if x != readme
+        and "/node_modules/" not in str(x)
+        and "/dist/" not in str(x)
+    ]
```

Comment:
```suggestion
    children = []
    for x in subdir.glob("**/*.md"):
      if x != readme and not ("/node_modules/" in str(x) or "/dist/" in str(x):
        children.append(x)
```

This just because of the difficulty of reading multi-line comprehension conditions

Reply:
Hmm, or moving the `not` (this is bikeshedding at this point):

```suggestion
    children = []
    for x in subdir.glob("**/*.md"):
      if not (x == readme or "/node_modules/" in str(x) or "/dist/" in str(x)):
        children.append(x)
```

Reply:
Factored out the predicate as a function.